FlowCrypt/Functionality/Services/Remote Pub Key Services/AttesterApi.swift

```diff
@@ -91,37 +92,18 @@ extension AttesterApi {
             body: pubkey.data(),
             headers: headers
         )
-        return Promise { () -> String in
-            let res = try awaitPromise(ApiCall.call(request))
-            return res.data.toStr()
-        }
-    }
-
-    @discardableResult
-    func replaceKey(email: String, pubkey: String) -> Promise<String> {
```

Comment:
While not used now, this will actually be needed later: on consumer usecases (that don't use the EKM), if the public key on Attester has different fingerprint than the public key being submitted, we'll have to use `replaceKey` instead of `updateKey`. I'll file an issue for this.

No need to add it back now, I can just copy/paste it below:

```
 @discardableResult
    func replaceKey(email: String, pubkey: String) -> Promise<String> {
        let request = ApiCall.Request(
            apiName: Constants.apiName,
            url: urlPub(emailOrLongid: email),
            method: .post,
            body: pubkey.data()
        )
        return Promise { () -> String in
            let res = try awaitPromise(ApiCall.call(request))
            return res.data.toStr()
        }
        let res = try await ApiCall.asyncCall(request)
        return res.data.toStr()
    }
```